webpack.config.babel.js

```diff
@@ -203,6 +204,10 @@ if (isProduction) {
         warnings: false,
       },
     }),
+    new CopyWebpackPlugin([
+      { from: 'packages/bpk-docs/src/Readme.md', to: 'Readme.md' },
```

Comment:
Does this get copied to the `dist` directory?

Reply:
yes it does!